assignments/objects.js

```diff
@@ -18,27 +18,45 @@ const example = {
 }
 
 // Write your intern objects here:
+const objMaker = (id, name, email, gender) => {
+  let result = {
```

Comment:
Great job here creating a object that accepts parameters that you can just pass in.  I believe the idea here was to get some exposure to writing objects.  This way, every time you call the object, you'll need to pass in parameters.